### ISSUE TITLE
Always set `<receive_untested_updates>false</receive_untested_updates>`

### ISSUE THREAD

**Issue by davidhedlund** [cemu-project/Cemu#1416]
This is relevant when you script cemu by only adding custom configurations, for example:

```
<?xml version="1.0" encoding="UTF-8"?>
<content>
    <fullscreen_menubar>true</fullscreen_menubar>
</content>
```

### Current Behavior

 `<receive_untested_updates>true</receive_untested_updates>` is added to settings.xml

### Expected Behavior

 `<receive_untested_updates>false</receive_untested_updates>` should be added to settings.xml

### Steps to Reproduce

* Open Cemu. It will generate `<receive_untested_updates>false</receive_untested_updates>` in settings.xml
* Close Cemu
* Remove all lines in settings.xml except `<?xml version="1.0" encoding="UTF-8"?>`
* Start Cemu again

**Comment by Exzap** [comment]
Fixed in 719c631f13c451e045903e79768e98e264a0e9b2

**Comment by davidhedlund** [comment]
> Fixed in [719c631](https://github.com/cemu-project/Cemu/commit/719c631f13c451e045903e79768e98e264a0e9b2)

I really appreciate it!
